app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,50 @@
+class CommentsController < ApplicationController
+
+  def index
+    @comments = Comment.all
```

Comment:
Should you be getting all the comments? Or all of the comments that belong to a particular post?